docs/cloud/overview.md

```diff
@@ -2,15 +2,15 @@
 
 Earthly Cloud is a collection of features that enrich the Earthly experience via cloud-based services. These include:
 
-* [Earthly Cloud Secrets](./cloud-secrets.md): A secret management system that allows you to store secrets in a cloud-based service and use them across builds.
 * [Earthly Satellites](./satellites.md): Cloud-based Buildkit instances managed by the Earthly team.
+* [Earthly Cloud Secrets](./cloud-secrets.md): A secret management system that allows you to store secrets in a cloud-based service and use them across builds.
 * [Earthly CI](./earthly-ci.md): A cloud-based CI/CD system that allows you to continuously build your code in the cloud.
```

Comment:
@vladaionescu Do we have any docs on log sharing? If we do (or if we could write some), that would be good to have here too.

Reply:
We do not, although, at the very least, we could mention this functionality in this list. I'll add that.